001-devcycle.conf.template

```diff
@@ -1,25 +1,11 @@
-WSGIPythonPath usr/local/devcycle
```

Comment:
One thing I'm just realizing now is that it looks like this previously ran the server as a WSGI daemon, which we're no longer doing. It may or may not make any difference, but we need to check that.